### ISSUE TITLE
Consider mirrorring dependencies

### ISSUE THREAD

**Issue by sebastianbergmann** [bebbo/amigaos-cross-toolchain#11]
At the time of writing I cannot download `http://sun.hasenbraten.de/vasm/release/vasm.tar.gz`. The HTTP request hangs forever.

I think it would be great if files that are not hosted on mirror networks such as gnu.org are mirrorred somewhere. GitHub comes to mind. Of course, the downloader needs to be changed then to download from the mirror.

**Comment by sebastianbergmann** [comment]
https://github.com/leffmann/vasm looks like a GitHub mirror of the vasm sources.

**Comment by sebastianbergmann** [comment]
https://github.com/leffmann/vlink looks like a GitHub mirror of the vlink sources.

**Comment by sebastianbergmann** [comment]
`sun.hasenbraten.de` is back online but at least one of the release tarballs hosted there (and downloaded by this toolchain's build) is broken at the moment:

```
$ wget wget http://sun.hasenbraten.de/vlink/release/vlink.tar.gz
```
```
$ tar xf vlink.tar.gz 
gzip: stdin: unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
```

**Comment by sebastianbergmann** [comment]
The (correct) files are back up.

**Comment by sebastianbergmann** [comment]
At the moment `mail.pb-owl.de` cannot be reached and `vbcc_target_m68k-amigaos.lha` cannot be downloaded.

**Comment by sebastianbergmann** [comment]
URL for `vbcc_target_m68k-amigaos.lha` (see https://github.com/bebbo/amigaos-cross-toolchain/issues/11#issuecomment-322756432) was fixed in 0ef7d79bd71292a6d3f18c8e2a2aef35731122e6.
